### ISSUE TITLE
KMeans tests fail with new version of scikit-learn

### ISSUE THREAD

**Issue by aulemahal** [Ouranosinc/xclim#317]
* xclim version: all
* Python version: all
* Operating System: all

### Description
Yesterday (3 dec 2019), scikit-learn 0.22 came out and since then, all tests of `TestEnsembleReduction` fail. The doc says the behaviour of `sklearn.cluster.KMeans` changed when called with `njobs=1`, which is the default and what we use.

### What I Did
Tested the same version of xclim only changing scikit-learn from 0.21.3 to 0.22, tests fails at 0.22.

The changelog: https://scikit-learn.org/stable/whats_new/v0.22.html

**Comment by aulemahal** [comment]
I found this tweak to help, simply add this `init` to the class where tests are failing so that it doesn't run those with older versions of `scikit-learn`.

```python
class TestEnsembleReduction:
    nc_file = os.path.join(TESTS_DATA, "EnsembleReduce", "TestEnsReduceCriteria.nc")

    def __init__(self):
        pytest.importorskip('sklearn', minversion='0.22')
```

**Comment by aulemahal** [comment]
Fixed by #318.
